wordpress-to-github/common/index.js

```diff
@@ -386,13 +442,16 @@ const syncBinaryFile = async (wordpress_url, gitRepo, mediaTree, endpoint) => {
  *
  * @param {*} json
  * @param {string[]} [excludeList]
+ * @returns {string} 
```

Comment:
Enters as an any/object but returns as a string?

Reply:
Good call, this is wrong. I fixed it up.